src/wasm.h

```diff
@@ -1158,6 +1158,19 @@ struct Importable {
   bool imported() { return module.is(); }
 };
 
+class Function;
+
+// Represents a mapping of wasm module elements to their location in the
+// binary representation. This is used for general debugging info support.
+// Offsets are relative to the beginning of the code section, as in DWARF.
+struct BinaryLocations {
+  struct Span {
+    uint32_t start, end;
```

Comment:
My thinking is that using `size_t` is risky as then it would behave differently on 32/64-bit host OSes. Using `int64_t` would be more future-proof, but takes twice the memory, which is expected to be a concern here. So I think `uint32_t` is the safe thing for now. But maybe it should be a typedef for easy changes in the future?